### PR TITLE
[#889] improvement: Modify default value of `rss.server.max.concurrency.of.per-partition.write`  to `30`.

### PR DESCRIPTION
**Pull request by leixm**
### What changes were proposed in this pull request?

Modify default value of `rss.server.max.concurrency.of.per-partition.write`  to `30`.

### Why are the changes needed?

For 0.8.0 Release.

### Does this PR introduce _any_ user-facing change?

No.

### How was this patch tested?

exiting UTs.

### CODE REVIEW

File: docs/server_guide.md [hunk 1]
```diff
@@ -144,7 +144,7 @@ rss.server.flush.localfile.threadPool.size 20
 rss.server.flush.hadoop.threadPool.size 20
```
Comment (zuston) [inline docs/server_guide.md:144]: This should be increased to > 30?

Comment (leixm) [reply docs/server_guide.md:144]: You're right, do you have a suggested value? how about we set it to 50?

Comment (leixm) [reply docs/server_guide.md:144]: Our production environment is set to 100.

Comment (zuston) [reply docs/server_guide.md]: > You're right, do you have a suggested value? how about we set it to 50?

2 * 30 = 60?

Comment (leixm) [reply docs/server_guide.md]: > > You're right, do you have a suggested value? how about we set it to 50?
> 
> 2 * 30 = 60?

Ok.

**Review by zuston — APPROVED** [review]
LGTM

**Pull request merged**